shared/actions/config/index.js

```diff
@@ -75,6 +76,8 @@ function isFollowing(getState: () => any, username: string): boolean {
 
 const waitForKBFS = (): AsyncAction => dispatch =>
   new Promise((resolve, reject) => {
+    dispatch(fuseStatus())
```

Comment:
Maybe move it here: https://github.com/keybase/client/blob/master/shared/actions/config/index.js#L163